docs/deployment-guide.md

```diff
@@ -880,7 +960,7 @@ You can trigger the reindexing of existing content in Search Enterprise using a
 pipenv run ansible-playbook playbooks/search-enterprise-reindex.yml -i <inventory_file>.yml
 ```
 
-## Cleanup & Uninstallation of ACS
+## Cleanup and uninstal ACS
 
```

Comment:
```uninstall```

Reply:
it looked weird to me but my grammar checker was not complaining so... xD